### PR TITLE
Cleanup redundant asciidoc formatting from clauses examples

### PR DESCRIPTION
**Pull request by stefano-ottolenghi**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-583.surge.sh

File: modules/ROOT/pages/clauses/index.adoc [hunk 1]
```diff
@@ -169,7 +169,7 @@ These comprise sub-clauses that must operate as part of reading clauses.
 m| xref::clauses/where.adoc[WHERE]
 | Adds constraints to the patterns in a `MATCH` or `OPTIONAL MATCH` clause or filters the results of a `WITH` clause.
 
-m| xref::clauses/order-by.adoc[+ORDER BY [ASC[ENDING] \| DESC[ENDING]]+]
+m| xref::clauses/order-by.adoc[ORDER BY [ASC[ENDING] \| DESC[ENDING]]]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/index.adoc]: This does not render properly 

Screenshot 2023-05-29 at 11 53 10

**Review by JPryce-Aklundh — APPROVED** [review]
Thanks a lot @stefano-ottolenghi ! I just noticed one thing that did not render as it should

File: modules/ROOT/pages/clauses/index.adoc (hunk 1 above)
Comment (stefano-ottolenghi) [reply modules/ROOT/pages/clauses/index.adoc]: Ah, good catch! The brackets needed escaping (already before, but the issue was hidden by the extra `+`).

**Pull request merged**